src/dnx/pal.win32.cpp

```diff
@@ -81,19 +83,56 @@ BOOL GetFullPath(LPCTSTR szPath, LPTSTR pszNormalizedPath)
     return TRUE;
 }
 
+namespace
+{
+    std::wstring get_runtime_path(TraceWriter& trace_writer)
+    {
```

Comment:
Added support for DNX_SERVICING. If the file does not exists or no entries are found in this file we will fall back and check the index.txt in the `%ProgramFiles%` if one exists.